### PR TITLE
Add some styling

### PR DESCRIPTION
**Pull request by mugishasam123**
- add some CSS styles to match the design

### CODE REVIEW

File: src/components/styles/bookCard.css [hunk 1]
```diff
@@ -1,8 +1,97 @@
...
+}
+
+.actions > button,
+.author {
+  color: #7bc0f5;
+  font-size: 17px;
+}
+
+.book-details {
+  display: flex;
```
Comment (Lameck1) [inline src/components/styles/bookCard.css]: - [ ] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS files. Please refactor it to a class and apply the class in the JSX to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to JSX element that needs this behavior:

```JSX
  <section className="d-flex">
```

File: src/components/styles/bookCard.css [hunk 2]
```diff
@@ -1,8 +1,97 @@
...
+  flex-direction: column;
+  gap: 6px;
+}
+
+.details {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
```
Comment (Lameck1) [inline src/components/styles/bookCard.css]: - [ ] Let's improve the styling of this section. The font-family for the category should be `Montserrat`. For the book title and the three buttons below the title, it should be `RobotoSlab` according to the design, please refactor this. Screenshots below will help show the differences in the font-family:

  | Design | You Implementation |
  | --- | --- |
  | image | image |

File: src/components/styles/form.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  padding: 16px;
+  font-size: 17px;
+}
+
+form > select {
+  width: 30%;
+  border: none;
+  padding: 16px;
+  color: grey;
+}
```
Comment (Lameck1) [inline src/components/styles/form.css]: - [ ] In this task we are to follow the provided design as strictly as possible and that includes colors, font faces, typography, e.t.c. Please implement the design for this input field and the select field as in the original design. The screenshot below will help show us the expectation:

  | The Design | Your implementation |
  | --- | --- |
  | image | image |

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @mugishasam123, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/form.css (hunk 3 above)
Comment (mugishasam123) [reply src/components/styles/form.css]: - this is how it looks on my screen,I think it matches the design


image

Comment (Lameck1) [reply src/components/styles/form.css]: I think you are good to go. I have realized pages are rendering with anomalies on my browser today. Sorry for this.

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @mugishasam123** , 

### **Congratulations! 🎉 Project Approved ✅**

Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**